src/computer.cpp

```diff
@@ -1393,31 +1393,16 @@ THE LOCAL OFFICE CAN BE REACHED BETWEEN THE HOURS OF 9AM AND \n\
 IF YOU WOULD LIKE TO SPEAK WITH SOMEONE IN PERSON OR WOULD LIKE\n\
 TO WRITE US A LETTER PLEASE SEND IT TO...\n" ) );
 
-    const tripoint your_pos = g->u.global_omt_location();
-    const tripoint center_pos = overmap_buffer.find_closest( your_pos, "evac_center_13", 0, false );
-
-    if( center_pos == overmap::invalid_tripoint ) {
-        query_any( _( "You don't know where the address could be..." ) );
-        return;
-    }
-
-    if( overmap_buffer.seen( center_pos.x, center_pos.y, center_pos.z ) ) {
-        query_any( _( "You already know that address..." ) );
-        return;
+    const mission_type_id &mission_type = mission_type_id( "MISSION_REACH_REFUGEE_CENTER" );
+    const std::vector<mission *> missions = g->u.get_active_missions();
+    if( !std::any_of( missions.begin(), missions.end(), [ &mission_type ]( mission * mission ) {
```

Comment:
Any hint why am I not having this compilation issue? This will improve my compilation-proofing